app/admin_components/adf-services/dreamfactory-services.js

```diff
@@ -348,6 +348,45 @@ angular.module('dfServices', ['ngRoute', 'dfUtility'])
                         scope.applyFilter();
                     }
                 });
+
+                var goToSelectedApiDoc = function () {
+                    
+                    if (!scope.dataLoading) {
+                        // We want to seperate out the service name, which can be done by pulling out anything
+                        // after the second hash.
+                        var url = top.window.location.hash,
+                        delimiter = '#',
+                        start = 2,
+                        serviceNameAsArray = url.split(delimiter).slice(start),
+                        selectedServiceName = serviceNameAsArray.toString();
```

Comment:
no need to create two variables if the serviceNameAsArray is not used somewhere else.

Just:
```
selectedServiceName = url.split(delimiter).slice(start).toString();
```
would be enough

Reply:
Done